nameserver/server.go

```diff
@@ -72,17 +146,16 @@ func rdnsHandler(config *dns.ClientConfig, lookups []Lookup) dns.HandlerFunc {
 	}
 }
 
-/* When we receive a request for a name outside of our '.weave.local.'
-   domain, ask the configured DNS server as a fallback.
-*/
-func notUsHandler(config *dns.ClientConfig) dns.HandlerFunc {
+// When we receive a request for a name outside of our '.weave.local.'
+// domain, ask the configured DNS server as a fallback.
+func (s *DNSServer) notUsHandler() dns.HandlerFunc {
```

Comment:
This seems like a gratuitous change; nothing in this function needs anything from DNSServer, so the code has become less general for no gain.